### PR TITLE
Owm email hidden

### PR DESCRIPTION
**Pull request by andibraeu**
preparation for https://github.com/freifunk-berlin/firmware-wizard-frontend/issues/33

### CODE REVIEW

File: utils/luci-app-owm/luasrc/owm.lua [hunk 1]
```diff
@@ -285,14 +285,19 @@ function get()
 		revision=version.distversion --owm
 	}
 
+
+	local email2owm = uci:get_first("freifunk", "owm", "email2owm")
```
Comment (andrenarchy) [inline utils/luci-app-owm/luasrc/owm.lua]: `email2owm` -> `submit_email`?

**Comment by andibraeu** [comment]
good point...

File: utils/luci-app-owm/luasrc/owm.lua [hunk 2]
```diff
@@ -285,14 +285,19 @@ function get()
 		revision=version.distversion --owm
 	}
 
+
+	local email2owm = uci:get_first("freifunk", "owm", "submit_email")
+	if not email2owm then
+		email2owm = '0'
+	end
```
Comment (andrenarchy) [inline utils/luci-app-owm/luasrc/owm.lua]: i think we can shorten these 4 lines to `local submit_email = tonumber(uci:get_first("freifunk", "owm", "submit_email")) or 0`

File: utils/luci-app-owm/luasrc/owm.lua [hunk 3]
```diff
@@ -285,14 +285,19 @@ function get()
...
 	root.freifunk = {}
 	uci:foreach("freifunk", "public", function(s)
 		local pname = s[".name"]
 		s['.name'] = nil
 		s['.anonymous'] = nil
 		s['.type'] = nil
 		s['.index'] = nil
-		if s['mail'] then
+		if s['mail'] and email2own == '1' then
```
Comment (andrenarchy) [inline utils/luci-app-owm/luasrc/owm.lua]: `if s['mail'] and submit_email == 1 then

**Comment by SvenRoederer** [comment]
There seems to be no checkbox in the UI to control this setting.

**Comment by booo** [comment]
Closing this due to inactivity.

**Pull request closed without merging**